controller/controller.go

```diff
@@ -220,6 +231,24 @@ func (c *controller) updateIngresses() error {
 						entry.BackendMaxConnections = tmp
 					}
 
+					if proxyBufferSizeString, ok := ingress.Annotations[proxyBufferSizeAnnotation]; ok {
+						tmp, _ := strconv.Atoi(proxyBufferSizeString)
+						entry.ProxyBufferSize = tmp
+						if tmp > maxAllowedProxyBufferSize {
+							log.Warnf("ProxyBufferSize %d has a value which exceeds the max permissible value. Resetting to: %d.", tmp, maxAllowedProxyBufferSize)
```

Comment:
I don't think resetting is the correct word here. It should tell the user that the value they have set exceeds `maxAllowedProxyBufferSize` and that it will use that value instead.

Reply:
Agreed. Changed as per suggestion